workflows/login.rb

```diff
@@ -4,8 +4,29 @@
 class Login < Workflow
     #probably implement a test-superclass, or use a framework for that?
 
+
+    # for example, this type of init can likely be standardized?
```

Comment:
Scheduling and issuing appointments would be basically two separate workflows chained together?

I think my next PR might be a way to pass results from one test to the next test in the workflow controller

So that you could basically pipe
schedule appointment # => yields appointmentnumber
Dispatch appointment # => requires appointment number

Reply:
This comment was removed. It's set up to have workflow as the superclass, with some standardized stuff there